src/components/FormEditCar/index.jsx

```diff
@@ -39,178 +50,220 @@ const FormEditCar = () => {
             onChange={handleChange}
-            placeholder="Seats"
+            placeholder="Car Name"
           />
         </div>
 
-      </div>
-
-      <div className="container__add-inputs">
-        <label className="add_label-title"><b>Luggage</b></label>
-        <br />
-        <div className="input-group">
-          <div className="container__span">
-            <span className="input-group-text">
-              <img src="https://res.cloudinary.com/dbmertsgv/image/upload/v1693273257/samples/ICONS_CAR_EDIT/luggage_vqprta.png" alt="luggage" />
-            </span>
-          </div>
+        <div className="container__add-inputs">
+          <label htmlFor="car-type" className="add_label-title"><b>Car Type</b></label>
+          <br />
           <input
             className="inputs__add"
-            type="number"
-            value={selectedCar.luggage}
-            name="luggage"
+            type="text"
+            value={updatedData.type}
+            name="type"
             onChange={handleChange}
-            placeholder="Luggage"
+            placeholder="Car Name"
           />
         </div>
 
-      </div>
+        <div className="container__add-inputs">
+          <span>
+            <b>Current Image</b>
+            <br />
+            <div className="container_img_update">
+              <img src={selectedCar.img} alt="" />
+            </div>
+          </span>
+        </div>
 
-      <div className="container__add-inputs">
-        <label className="add_label-title"><b>Air Conditioner</b></label>
-        <br />
-        <div className="input-group">
-          <div className="container__span">
-            <span className="input-group-text">
-              <img src="https://res.cloudinary.com/dbmertsgv/image/upload/v1693273273/samples/ICONS_CAR_EDIT/snowflake_uqwwpz.png" alt="air" />
-            </span>
+        <div className="container__add-inputs">
+          <label className="add_label-title">
+            <b>Upload Car Image here</b>
+          </label>
+          <br />
+          <div className="submit_file">
+            <div className="section__submit_file">
+              <input
+                type="file"
+              />
+              <i className="icon-cloud-up" />
+              <h4><b>Drop car img here or click to upload.</b></h4>
+            </div>
           </div>
+        </div>
 
-          <select id="car-type" className="select_add">
-            <option>Please choose an option</option>
-            <option>True</option>
-            <option>False</option>
-          </select>
+        <div className="container__add-inputs">
+          <label className="add_label-title"><b>Seats</b></label>
+          <br />
+          <div className="input-group">
+            <div className="container__span">
+              <span className="input-group-text">
+                <img src="https://res.cloudinary.com/dbmertsgv/image/upload/v1693273273/samples/ICONS_CAR_EDIT/seat_e37og3.png" alt="seat" />
```

Comment:
Please, move this img url to a constant that could be imported and set here.

Reply:
Hi, Hector! We're going to put this on the backlog we'll do it with all the UI images that comes from Cloudinary. 👍